server/app/src/test/java/com/google/heartrate/wearos/app/gatt/heartrate/characteristics/HeartRateGattCharacteristicManagerTest.java

```diff
@@ -0,0 +1,159 @@
+package com.google.heartrate.wearos.app.gatt.heartrate.characteristics;
+
+import android.os.Build;
+
+import com.google.heartrate.wearos.app.gatt.CharacteristicsArgumentProvider;
+import com.google.heartrate.wearos.app.gatt.GattException;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.RobolectricTestRunner;
+import org.robolectric.annotation.Config;
+
+import static com.google.heartrate.wearos.app.gatt.CharacteristicsArgumentProvider.Range.IN_UINT16;
+import static com.google.heartrate.wearos.app.gatt.CharacteristicsArgumentProvider.Range.IN_UINT8;
+import static com.google.heartrate.wearos.app.gatt.CharacteristicsArgumentProvider.Range.OUT_UINT16;
+import static com.google.heartrate.wearos.app.gatt.TestUtils.assertNotThrows;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThrows;
+
+@RunWith(RobolectricTestRunner.class)
+@Config(sdk = {Build.VERSION_CODES.O_MR1}, manifest = Config.NONE)
+public class HeartRateGattCharacteristicManagerTest {
```

Comment:
HeartRateGattCharacteristicManagerTest -> HeartRateMeasurementCharacteristicTest ?

Reply:
Renamed in [commit](https://github.com/googleinterns/heartrate-bt-wear/pull/1/commits/93a29ed3ea6d7a2bf9235ec829a41d0e44889a43)